src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
```

Comment:
no needs to add class selector to a body. you can reset default body margin using tag selector
```suggestion
  <body>
```